packages/glob/src/internal-glob-options-helper.ts

```diff
@@ -0,0 +1,32 @@
+import * as core from '@actions/core'
+import {GlobOptions} from './internal-glob-options'
+
+/**
+ * Returns a copy with defaults filled in.
+ */
+export function getOptions(copy?: GlobOptions): GlobOptions {
```

Comment:
this function basically just moved from internal-pattern-helper.ts

also the default for followSymbolicLinks flipped to false

Reply:
true or false?

Reply:
flipped back to true!